tests/test_claude_hook_clear_running_status.py

```
@@ -127,62 +142,3592 @@ def _response_for(self, line: str) -> str:
+        surface_id=surface_id,
+    ) as server:
+        state_path = Path(server.root.name) / "retired-source-state.json"
+        env = hook_environment(server, workspace_id, surface_id, state_path)
+        run_claude_hook(
+            cli_path,
+            server.socket_path,
+            "prompt-submit",
+            {
+                "session_id": retired_session_id,
+                "turn_id": "pre-clear-turn",
+                "cwd": "/tmp",
```

Ruff (0.16.0)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)